### DISCUSSION TITLE
Realearn with rosetta in reaper doesn't work

### DISCUSSION THREAD

**Discussion by mike-not-weird** [helgoboss/helgobox#1265] [General]
I programmed a few of my midi controllers with Realearn on my Mac M1. I learned that I had to use Reaper with Rosetta because some of my plugins wouldn't work otherwise. When I start Reaper with Rosetta my Realearn my configurations can't be loaded anymore. How can I use realearn in this environment?

**Comment by helgoboss** [comment]
I think you need to uninstall ReaLearn for ARM and reinstall it for x64. You can only have one.

**Comment by mike-not-weird** [comment]
In fact I had to uninstall, Realearn, SWS and Reapack ARM libraries and reinstall their « X86_64 » version.

Now everything works fine. Thanks.
